packages/cbioportal-clinical-timeline/src/TimelineStore.ts

```diff
@@ -49,6 +51,15 @@ export class TimelineStore {
         setTimeout(this.setScroll.bind(this), 10);
     }
 
+    setGroupByOption(value: string) {
```

Comment:
since this mutates observable state, it should be given @action decorator.   this is a convention which allows bulk update (when necessary). in this case, it doesn't really do anything, but still worth abiding by convention

Reply:
fixed